### PR TITLE
feat: broadcast work time changes on the owner's user channel

### PR DESCRIPTION
**Pull request by patrickweh**
When a user changes a work time, the change now also broadcasts on that **user's** private channel, so their own other sessions (e.g. the Nuxbe Desktop client and an open web app) refresh live. Other users are unaffected.

## Details
- `WorkTime::broadcastOn()` adds a `PrivateChannel` for the owning user (`user.{user_id}`) alongside the existing model channels.
- The `WorkTime` Livewire component reloads its daily work time / pause on the `WorkTimeCreated`/`WorkTimeUpdated`/`WorkTimeDeleted` events on the current user's channel. Its load logic is extracted into `loadWorkTimes()` and now resets the forms before filling, so an ended day clears correctly.

## Tests
- A work time broadcasts on the owning user channel.
- It does not broadcast on another user's channel.
- A work time without a user is not broadcast on a user channel.

## Summary by Sourcery

Broadcast work time changes on the owning user’s private channel and update the Livewire WorkTime component to reload daily work times and pauses in response to these events.

New Features:
- Broadcast work time model events on the owning user’s private channel so the user’s other sessions stay in sync.

Bug Fixes:
- Reset and reload daily work time and pause forms when work times are refreshed so ended days clear correctly.

Enhancements:
- Refactor the Livewire WorkTime component to extract work time loading into a reusable loadWorkTimes() method.
- Extend WorkTime model broadcasting to include user-specific private channels when a user_id is present.

Tests:
- Add unit tests verifying work time events broadcast only on the owning user’s channel and not on other or null-user channels.

### CODE REVIEW

File: src/Livewire/WorkTime.php [hunk 1]
```diff
@@ -263,9 +271,14 @@ public function toggleWorkDay(bool $start): void
 
     protected function getListeners(): array
     {
+        $userChannel = 'echo-private:' . app(User::class)->getMorphClass() . '.' . auth()->id();
```
Comment (SirSplasch) [inline src/Livewire/WorkTime.php]: Use morph_alias() helper function instead of app()->getMorphClass()

File: src/Models/WorkTime.php [hunk 2]
```diff
@@ -258,6 +260,19 @@ public function employeeDays(): BelongsToMany
             ->withTimestamps();
     }
 
+    public function broadcastOn(string $event): array
+    {
+        $channels = Arr::wrap(parent::broadcastOn($event));
+
+        if (! is_null($this->user_id)) {
+            $channels[] = new PrivateChannel(
+                app(User::class)->getMorphClass() . '.' . $this->user_id
```
Comment (SirSplasch) [inline src/Models/WorkTime.php]: morph_alias()

**Review by SirSplasch — CHANGES_REQUESTED** [review]

**Review by SirSplasch — APPROVED** [review]
